packages/frontend/src/components/common/ResourceView/ContentTypeMultiSelect.tsx

```diff
@@ -0,0 +1,94 @@
+import { ContentType } from '@lightdash/common';
+import { Group, MultiSelect, Text } from '@mantine/core';
+import {
+    IconChartBar,
+    IconLayoutDashboard,
+    type Icon,
+} from '@tabler/icons-react';
+import React, { forwardRef, type FC } from 'react';
+import MantineIcon from '../MantineIcon';
+
+const ContentTypeOptions = [
+    {
+        value: ContentType.DASHBOARD,
+        label: 'Dashboard',
+        colour: 'green.8',
+        icon: IconLayoutDashboard,
+    },
+    {
+        value: ContentType.CHART,
+        label: 'Chart',
+        colour: 'blue.8',
+        icon: IconChartBar,
+    },
+];
+
+interface ItemProps extends React.ComponentPropsWithoutRef<'div'> {
+    label: string;
+    colour: string;
+    icon: Icon;
+}
+
+const ContentTypeSelectOption = forwardRef<HTMLDivElement, ItemProps>(
+    ({ label, icon, colour, ...others }: ItemProps, ref) => (
+        <div ref={ref} {...others}>
+            <Group spacing={'xs'}>
+                <MantineIcon
+                    icon={icon}
+                    fillOpacity={0.1}
+                    fill={colour}
+                    color={colour}
```

Comment:
color=colour 😄

Change or not, this is very local

Reply:
fixed 👍